src/plugins/expressions/public/index.ts

```diff
@@ -122,4 +122,7 @@ export {
   TypeToString,
   UnmappedTypeStrings,
   ExpressionValueRender as Render,
+  ExpressionsService,
+  ExpressionsServiceSetup,
+  ExpressionsServiceStart,
```

Comment:
Do these need to be exported for the docs to work? Because otherwise I don't see a need for them as folks should use `ExpressionsSetup` and `ExpressionsStart` which extend from them

Reply:
yeah, we need to, everything we extend should be exported as well.